programs/locfile.c: In function `locfile_read':
programs/locfile.c:863: error: label at end of compound statement
make[2]: *** [/home/dank/wk/crosstool-0.28-rc4h/build/i686-unknown-linux-gnu/gcc-3.4.0-20040406-glibc-2.1.3/build-glibc/locale/locfile.o] Error 1
make[2]: Leaving directory `/home/dank/wk/crosstool-0.28-rc4h/build/i686-unknown-linux-gnu/gcc-3.4.0-20040406-glibc-2.1.3/glibc-2.1.3/locale'

programs/locale.c: In function `print_item':
programs/locale.c:666: error: label at end of compound statement
make[2]: *** [/home/dank/wk/crosstool-0.28-rc4h/build/i686-unknown-linux-gnu/gcc-3.4.0-20040406-glibc-2.1.3/build-glibc/locale/locale.o] Error 1
make[2]: Leaving directory `/home/dank/wk/crosstool-0.28-rc4h/build/i686-unknown-linux-gnu/gcc-3.4.0-20040406-glibc-2.1.3/glibc-2.1.3/locale'


Mechanical changes to quiet the errors.

--- glibc-2.1.3/locale/programs/locfile.c.old	Sun Apr 11 20:03:52 2004
+++ glibc-2.1.3/locale/programs/locfile.c	Sun Apr 11 20:04:47 2004
@@ -860,6 +860,7 @@
 
 	    collate_end_weight (ldfile, result);
 	  illegal_weight:
+		;	/* gcc-3.4 requires a statement here */
 	  }
 	  continue;
 
--- glibc-2.1.3/locale/programs/locale.c.old	Sun Apr 11 20:08:48 2004
+++ glibc-2.1.3/locale/programs/locale.c	Sun Apr 11 20:09:04 2004
@@ -663,6 +663,7 @@
 	  }
 	  break;
 	default:
+		;       /* gcc-3.4 requires a statement here */
 	}
       putchar ('\n');
     }
